docs/hugo/content/tutorials/upgrade-resource-version/_index.md

```diff
@@ -0,0 +1,677 @@
+---
+title: "Upgrading a resource"
```

Comment:
This is going alongside "using aso tutorials" which seems a bit weird to me.

Do we maybe want a folder in `contributing` called `tutorials` as well, and then put this there?